templates/traas.yaml

```diff
@@ -54,6 +54,11 @@ parameters:
     default: CentOS-7-x86_64-GenericCloud-1503
     description: Image to boot as the undercloud instance
 
+  volume_id:
+    description: An external volume ID for docker images
```

Comment:
i would just call this an external volume ID, since it does not have to be just for docker images